### PR TITLE
Docs/i18n split

### PR DESCRIPTION
**Pull request by m0ksem**
closes https://github.com/epicmaxco/vuestic-ui/issues/2254
closes https://github.com/epicmaxco/vuestic-ui/issues/2970

Split translations for docs.
image

And testing docs in one PR:
image

### CODE REVIEW

**Comment by m0ksem** [comment]
Maybe gonna add a TS support here and store translations in one file after discussion.

**Comment by m0ksem** [comment]
Decided not to make a TS support and prefer separated files instead.

File: packages/docs/locales/en.json [hunk 1]
```diff
@@ -1667,7 +1667,8 @@
     "i18n": "I18n",
```
Comment (asvae) [inline packages/docs/locales/en.json:1667]: Some translations look to be broken:

image

**Review by asvae — CHANGES_REQUESTED** [review]
Local version is somewhat broken.

**Pull request merged**